src/main/java/io/epirus/console/Runner.java

```diff
@@ -49,12 +51,14 @@
 
     public static void main(String[] args) throws Exception {
         System.out.println(LOGO);
-        CliConfig config = CliConfig.getConfig(CliConfig.getDefaultEpirusConfigPath().toFile());
-        Updater updater = new Updater(config);
-        updater.promptIfUpdateAvailable();
-        Thread updateThread = new Thread(updater::onlineUpdateCheck);
+        config.setPersistent(true);
```

Comment:
this looks bad

Reply:
i see now - please use `Singleton.getInstance()` to make this clearer

Reply:
https://www.geeksforgeeks.org/singleton-class-java/

Reply:
I agree, will take this to slack as it's a weird situation.

Reply:
It's not quite a singleton, but it's better than it was.